src/dotnet/Telemetry.cs

```diff
@@ -19,6 +20,8 @@ public class Telemetry : ITelemetry
         private Dictionary<string, double> _commonMeasurements = null;
         private Task _trackEventTask = null;
 
+        internal static readonly string TelemetrySentinel  = Path.Combine(ApplicationEnvironment.ApplicationBasePath, $"{Product.Version}.dotnetTelemetry");
```

Comment:
I don't think you can garuntee write permission at ApplicationBasePath.  What if someone extracts to programfiles but then runs as regular user?  Shouldn't we be using either ProgramData or UserProfile/Home for this?